plasmapy/formulary/parameters.py

```diff
@@ -553,43 +685,47 @@ def thermal_speed(
     T: u.K,
     particle: Particle,
     method="most_probable",
-    mass: u.kg = np.nan * u.kg,
+    mass: u.kg = None,
     ndim=3,
 ) -> u.m / u.s:
     r"""
-    Return the most probable speed for a particle within a Maxwellian
-    distribution.
+    Calculate the speed of thermal motion for particles with a Maxwellian
+    distribution.  (See the :ref:`Notes <thermal-speed-notes>` section for
+    details.)
+
+    **Aliases:** `~plasmapy.formulary.parameters.vth_`
 
-    **Aliases:** `vth_`
+    **Lite Version:** `~plasmapy.formulary.parameters.thermal_speed_lite`
 
     Parameters
     ----------
     T : `~astropy.units.Quantity`
-        The particle temperature in either kelvin or energy per particle
+        The temperature of the particle distribution, in units of kelvin or
+        energy.
```

Comment:
I'm reluctant to add "per particle" here because temperature is generally proportional to the average energy per particle but not equal.  They're equivalent concepts but not equal.